prow/labels/labels.go

```diff
@@ -0,0 +1,24 @@
+package labels
+
+const LGTMLabel = "lgtm"
+const BlockedPathsLabel = "do-not-merge/blocked-paths"
+const LifecycleActiveLabel = "lifecycle/active"
+const LifecycleFrozenLabel = "lifecycle/frozen"
+const LifecycleStaleLabel = "lifecycle/stale"
+const LifecycleRottenLabel = "lifecycle/rotten"
+const ClaYesLabel = "cncf-cla: yes"
+const ClaNoLabel = "cncf-cla: no"
+const ApprovedLabel = "approved"
+const InvalidOwnersLabel = "do-not-merge/invalid-owners-file"
+const CpUnapprovedLabel = "do-not-merge/cherry-pick-not-approved"
+const CpApprovedLabel = "cherry-pick-approved"
+const WorkInProgressLabel = "do-not-merge/work-in-progress"
+const EvilCodeLabel = "do-not-merge/evil-code"
```

Comment:
This is just for a unit test. It doesn't need to be an exported label.

Please remove all the constants you created that are string literals in test files (like kind/bug, sig/awesome)

Reply:
Just removed the EvilCodeLabel and AwesomeLabel. I didn't remove kind/bug BugLabel though because it was also used in `prow/plugins/sigmention/sigmention.go`